lib/score.rb

```diff
@@ -0,0 +1,113 @@
+require_relative 'badscore_error'
+
+class Score
+  attr_reader :name, :scores
+
+  def initialize name = 'Sisyphus'
+    @name = name   
+    init_scores()
+  end
+
```

Comment:
I'm wondering whether a couple more classes might help improved readability - we could make it more intuitive than score.name giving the player's name. A user class could help here?